### ISSUE TITLE
1-org deployment - After pushing the code for 1-org on to plan and prod branch, nothing is deployed no build triggers

### ISSUE THREAD

**Issue by Atul7696** [terraform-google-modules/terraform-example-foundation#725]
### TL;DR

1-org deployment - After pushing the code for 1-org on to plan and prod branch, nothing is deployed no build triggers

Have deployed 0-bootstrap successfully from GCE instance with tf version 0.13.7 and cloud sdk. 

### Expected behavior

01-org is should be deployed via cloud build in cicd project but its not happening.

### Observed behavior

01-org is not deploying the shared folder structure via cloud build. 

Source repos and artifact repo and everything else is fine. Does 0-bootstrap setup triggers as well in cloud build? Stuck in deployment for 01-org. Seems I am missing something. Have followed the documentation and saw no errors. 

### Terraform Configuration

```hcl
Its same as from the 1-org with changes done in terraform.tfvars and variables.tf w.r.t to org id, svc account retrieved from previous step and billing account etc.
```

### Terraform Version

```sh
0.13.7
Using Ubuntu VM 18.0 with cloud sdk default version above the pre-requisite one.
Project quota has already been increased last week thats why was able to deploy 0-bootstrap.

Can somebody advise here?
```

### Additional information

In case you need any other information, please suggest and i shall share

**Comment by daniel-cit** [comment]
Hi @Atul7696

I would double check each step:

- check the output of step 0-bootsrap for:
  - `cloudbuild_project_id`,
  - `csr_repos -> gcp-org -> url`

check the cloud build triggers definition in the console:
https://console.cloud.google.com/cloud-build/triggers?project=<cloudbuild_project_id>

and the cloud build triggers history:
https://console.cloud.google.com/cloud-build/builds?project=<cloudbuild_project_id>

you should see the builds that were executed based in the steps in the README
https://github.com/terraform-google-modules/terraform-example-foundation/tree/master/1-org#deploying-with-cloud-build

in each item in the history you can check the build results to see if there is an error:
https://cloud.google.com/build/docs/view-build-results

**Comment by Atul7696** [comment]
Thanks, able to check from gcloud, able to deploy foundation till 3-networks. For 4-projects, will do later. Closing this issue.

**Comment by Atul7696** [comment]
Thanks, able to check from gcloud, able to deploy foundation till 3-networks. For 4-projects, will do later. Closing this issue
